lib/vdsm/virt/vmdevices/storage.py

```diff
@@ -686,6 +686,25 @@ def iotune(self, value):
         vmtune.validate_io_tune_params(iotune)
         self._iotune = iotune
 
+    def volume_path(self, vol_id):
```

Comment:
Not needed since we should pass uuids to storage. The storage layer knows how to locate the volume path.